ts/Extensions/ParallelCoordinates.ts

```diff
@@ -691,8 +690,33 @@ namespace ParallelAxis {
                 }
             });
 
-            axis.dataMin = arrayMin(currentPoints);
-            axis.dataMax = arrayMax(currentPoints);
+            // Take into account range series points as well (#15752)
+            let min: number = NaN,
+                max: number = NaN;
+
+            currentPoints.forEach(function (point): void {
+                if (Array.isArray(point)) {
```

Comment:
In Highcharts, we're using `Highcharts.isArray` (from Utilities) instead of `Array.isArray`